ui/src/shared/components/DatabaseDropdown.js

```diff
@@ -43,12 +43,18 @@ class DatabaseDropdown extends Component {
     const proxy = source.links.proxy
     try {
       const {data} = await showDatabases(proxy)
-      const {databases} = showDatabasesParser(data)
+      const {databases, errors} = showDatabasesParser(data)
+      if (errors.length > 0) {
+        throw errors[0] // only one error can come back from this, but it's returned as an array
```

Comment:
If an error comes back, what do you think about displaying it to the user via an alert notification, by passing it to the action creator `errorThrown`, rather than just the console?